### ISSUE TITLE
503 error after fresh install

### ISSUE THREAD

**Issue by mightydok** [modxcms/revolution#13438]
### Summary
503 error at main page after install

### Step to reproduce
Fresh install on centos 6 with ispconfig 3
install modx
Open main page and get 503 error with errors in error log file:
```
[2017-04-24 00:03:15] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:03:24] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:16] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:18] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:47] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:48] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:48] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:48] (FATAL) The requested resource has no valid content type specified.
[2017-04-24 00:16:48] (FATAL) The requested resource has no valid content type specified.
```
Manager console works fine

### Observed behavior
Get 503 error: Site temporarily unavailable.

### Expected behavior
Main page opened without error

### Environment
Centos 6
ISPConfig 3
Apache + php 5.4 Fastcgi
Mysql 5.5

**Comment by OptimusCrime** [comment]
Why close @mightydok ?

**Comment by OptimusCrime** [comment]
Closed in favor of #13439 by author. Posting this for reference.
